wasmcc/wasmcc.go

```diff
@@ -396,70 +388,69 @@ func txnResult(vmExecResult int64, resultGlobal []byte) pb.Response {
 	if vmExecResult == -1 {
 
 		//Check if some result is returned by wasm chaincode
-		if resultGlobal ==nil{
+		if resultGlobal == nil {
 			return shim.Error(strconv.FormatInt(vmExecResult, 10))
-		}else {
+		} else {
 			return shim.Error(string(resultGlobal))
 		}
-	}else{
-		if resultGlobal ==nil{
+	} else {
+		if resultGlobal == nil {
 			return shim.Success([]byte(strconv.FormatInt(vmExecResult, 10)))
-		}else {
+		} else {
 			return shim.Success(resultGlobal)
 		}
 	}
 }
 
-func decodeReceivedWASMChaincode(encodedChaincode string) ([]byte,error) {
+func decodeReceivedWASMChaincode(encodedChaincode string) ([]byte, error) {
 
 	var chaincodeDecoded []byte
 
 	//For hex string passed through cli
-	if strings.HasPrefix(encodedChaincode,"0061736d01000000"){
+	if strings.HasPrefix(encodedChaincode, "0061736d01000000") {
 		logger.Infof("Hex encoded wasm chaincode string")
 
 		logger.Debugf("Encoded wasm chaincode: " + encodedChaincode)
 		chaincodeDecoded, _ = hex.DecodeString(encodedChaincode)
-	}else{
+	} else {
 		decodedBytesTemp := []byte(encodedChaincode)
 		kind, _ := filetype.Match(decodedBytesTemp)
 		if kind == filetype.Unknown {
 			logger.Errorf("Unknown file type")
-			return nil,errors.New("unknown file type")
-		}else if kind.Extension == "wasm" {
+			return nil, errors.New("unknown file type")
+		} else if kind.Extension == "wasm" {
 			logger.Infof("wasm file")
 			chaincodeDecoded = decodedBytesTemp
-		}else if kind.Extension == "zip" {
+		} else if kind.Extension == "zip" {
 			logger.Infof("zip compressed file")
 
 			zipReader, err := zip.NewReader(bytes.NewReader(decodedBytesTemp), int64(len(decodedBytesTemp)))
 			if err != nil {
-				return nil,errors.New(err.Error())
+				return nil, errors.New(err.Error())
 			}
 
 			if len(zipReader.File) > 1 || len(zipReader.File) == 0 {
-				return nil,errors.New("error: more than one wasm file in zip archive")
+				return nil, errors.New("error: more than one wasm file in zip archive")
```

Comment:
I'm kind of curious about the multiple wasm case. Might be a future feature to support.